fastly/resource_fastly_service_v1_test.go

```diff
@@ -610,6 +610,10 @@ func TestAccFastlyServiceV1_defaultHost(t *testing.T) {
 				Config: testAccServiceV1Config_default_host(name, domain, defaultHost),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckServiceV1Exists("fastly_service_v1.foo", &service),
+					resource.TestCheckResourceAttr(
+						"fastly_service_v1.foo", "default_ttl", "3400"),
```

Comment:
```suggestion
						"fastly_service_v1.foo", "default_ttl", "3600"),
```

our default is 3600, not 3400 (considering `testAccServiceV1Config_default_host` config is not setting `default_ttl`)